pkg/resource/logging-config/promtail-logging-config.go

```diff
@@ -35,6 +35,7 @@ type promtailExtraEnv struct {
 }
 
 type promtailConfigSnippets struct {
+	PipelineStages      string               `yaml:"pipelineStages" json:"pipelineStages"`
```

Comment:
Using this https://github.com/grafana/helm-charts/blob/614fe8997c2a2cdd6e63b2c0e8816e76b875711f/charts/promtail/values.yaml#L504 for kubernetes pod scrape job